### PR TITLE
Campaign  member faming

### PR DESCRIPTION
**Pull request by weerd**
Resolves [DOS-61](https://jira.dosomething.org/browse/DOS-61)

@DFurnes Most of the promotions stuff will get moved into Neue later this week in a cleanup phase ;)

@aaronschachter Pinging you in case you wanna checkout the preprocess stuffz!

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc [hunk 1]
```diff
@@ -252,6 +256,23 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
...
     }
+
+    // Output sponsors and creator in promotions module
+    if ($vars['sponsor_logos'] || $vars['campaign_creator']) {
+      $classes = NULL;
+
+      if ($vars['sponsor_logos'] && $vars['campaign_creator']) {
+        $classes = ' -dual';
+        $vars['promotions'] = $vars['sponsor_logos'] . $vars['campaign_creator'];
+      } elseif ($vars['sponsor_logos']) {
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: newline here. https://www.drupal.org/coding-standards#controlstruct

File: lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc [hunk 2]
```diff
@@ -252,6 +256,23 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
...
+        $vars['promotions'] = $vars['sponsor_logos'];
+      } elseif ($vars['campaign_creator']) {
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: same

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.heading {
+  font-family: 'Proxima Nova', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss]: You can use the [`$font-proxima-nova`](https://github.com/DoSomething/neue/blob/aeb18096c75e9d0a49b9ceca968d9d029d746179/scss/helpers/_variables.scss#L32) stack for this.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss]: This should also be inherited from the `body` element, right? Do we need to redefine it here?

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 4]
```diff
@@ -8,6 +8,8 @@
  * - $classes: Additional classes passed for output (string).
  * - $campaign_creator: HTML for the Campaign Creator link/modal.
  */
+
+// dpm($variables);
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: NOOOOO Y U NO USE GIT PRE COMMIT HOOKS #2861

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-creator.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  }
+
+  .__member {
+    background-color: $yellow;
+    position: relative;
+
+    @include media($tablet) {
+      display: table-cell;
+      // float: left;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-creator.scss]: Drop this commented line?

**Comment by aaronschachter** [comment]
Bender approved pending cleanup comments.

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-creator.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+      }
+    }
+  }
+
+  .__body {
+    padding: 18px;
+
+    @include media($tablet) {
+      display: table-cell;
+      // float: left;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-creator.scss]: Drop commented out line?

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss (hunk 3 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss]: Setting up the precedent for the new "heading" module stuff we discussed w/ Crystal ;)

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php (hunk 4 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: Dammit. I thought I removed all those.

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_promotion.scss [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+            right: 10px;
+          }
+        }
+      }
+    }
+  }
+}
+
+
+// // Promotion Individual
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_promotion.scss]: Double comment? :rainbow: :rainbow:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss (hunk 3 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_heading.scss]: Totally. But is there any case where the headings wouldn't be inheriting the `font-family` property [defined in the body](https://github.com/DoSomething/neue/blob/aeb18096c75e9d0a49b9ceca968d9d029d746179/scss/_main/_base.scss#L16)?

**Comment by DFurnes** [comment]
Looks cool other than the wee little comments. :cake:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_promotion.scss (hunk 7 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_promotion.scss]: All the way! Probably from when I had a full selection of code I commented out in one shot :goat:

**Pull request merged**